src/facades/refreshToken.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+const { getById, db } = require('../lib/db')
+const uuid = require('uuid/v4')
+
+const REFESH_TOKEN_KEY_PREFIX = 'refreshTokenByUserId'
+
+/**
+ * Will return a refreshToken by email.
+ * Accepts one url param: email
+ * Matching is case sensitive and uses a startswith comparison
+ * @param params
+ */
+exports.getRefreshToken = async (userId) => {
+  return await getById(REFESH_TOKEN_KEY_PREFIX, userId)
+}
+
+exports.putRefreshToken = (userId) => {
+  const token = uuid()
```

Comment:
BTW this doesn't have to be a `UUID` it could be any random token for example 32 characters long.